src/trimmer_decorator.rb

```diff
@@ -0,0 +1,9 @@
+require_relative 'base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  MAX_LENGTH = 10
+
+  def correct_name
+    super[0, MAX_LENGTH]
+  end
```

Comment:
- [ ] Great job working on the required changes but I think it would be great if you add the condition that the output of @nameable.correct_name has a maximum of 10 characters. If it's longer it should trim the word. so if the word longer than 10 chars so trim the word and if it is not just return the word ⭐ so please add the condition 🤩⭐🚀 

- If you want to start from zero so you should finish with 9 to have a maximum 10 characters 🤩

![image](https://github.com/skabeo/school-library/assets/58553711/5eb8a3a7-23ab-450e-8d28-b0a28b59b924)